packages/css/src/types.ts

```diff
@@ -1,15 +1,24 @@
 import type { MapLeafNodes, CSSVarFunction } from '@vanilla-extract/private';
-import type { PropertiesFallback, AtRule, Properties } from 'csstype';
+import type { Properties, AtRule } from 'csstype';
 
 import type { SimplePseudos } from './simplePseudos';
 
-type CSSTypeProperties = PropertiesFallback<number | (string & {})>;
+// csstype is yet to ship container property types as they are not in
+// the output MDN spec files yet. Remove this once that's done.
+interface ContainerProperties {
+  container?: string;
+  containerType?: 'size' | 'inline-size' | 'block-size' | (string & {});
+  containerName?: string;
+}
+
+type CSSTypeProperties = Properties<number | (string & {})> &
```

Comment:
Why is `PropertiesFallback` no longer used here?

Reply:
I was having type issues, but then realised we'd already created out own fallback types to better support our CSS variable type so it was redundant to use the built in one. 

Have a look at the CSSProperties type just below.

Reply:
Ah yeah, I didn't catch that. Nice one.

Reply:
Is it worth adding a dummy type here with `@ts-expect-error` referencing `container` on `csstype`?

That way when we its available and we bump `csstype` we'll be notified to remove our version.